### ISSUE TITLE
MEET (IEEE/CAA JOURNAL OF AUTOMATICA SINICA. 2025)

### ISSUE THREAD

**Issue by pengfei-gao** [zytx121/Awesome-RS-VL-Data#84]
[![](https://img.shields.io/badge/Paper-20B2AA?style=for-the-badge)](https://www.ieee-jas.net/en/article/doi/10.1109/JAS.2025.125324) [![](https://img.shields.io/badge/Download-20B2AA?style=for-the-badge)](https://jerrywyn.github.io/project/MEET.html)

## 📌 Overview
**Source (1)**: MEET dataset (Fine-Grained geospatial scene classification)
**Total Samples**:  1.03 million
**Image Size**: 
**Data Usage**:  Training

**Source (2)**: UFZ (urban functional zone mapping) evaluation dataset
**Total Samples**:  4323
**Image Size**: 
**Data Usage**:  Evaluation 

## 📌 Data Pipeline
Existing approaches often rely on manually zooming remote sensing images at different scales to create typical scene samples. This approach fails to adequately support the fixed-resolution image interpretation requirements in realworld scenarios. To address this limitation, we introduce the million-scale fine-grained geospatial scene classification dataset (MEET), which contains over 1.03 million zoom-free remote sensing scene samples, manually annotated into 80 fine-grained categories. The images are collected from 2018 to 2022, with each sample containing a center scene with 256 × 256 pixels, along with a surrounding scene with 768 × 768 pixels and a global scene with 1280 ×1280 pixels. The overall distribution and some samples are
shown in Fig. 5. It is important to note that the spatial resolution of all samples is consistently set to 1.0 m.
OSM provides semantic annotations by labeling geographical features and land-use within its maps, offering detailed information about roads, buildings, and other points of interest. Subsequently, we preprocess the acquired data by performing coarse filtering or integration of scene categories based on the quality of OSM annotations, and design a series of rules to reduce noise at the image-level labels. Finally, we ensure a global distribution and high richness of samples by defining random spatial windows using geographic coordinates.
We randomly divide the entire dataset into training, valida tion, and testing sets by category, with a ratio of 6 : 2 : 2. More specifically, the training set, validation set, and testing set contain 620 237 206 755, and 206 755 samples, respectively.
In MEET, each scene sample follows a scene-in-scene layout, where the central scene serves as the reference, and auxiliary scenes provide crucial spatial context for fine-grained classification.

Across five cities (e.g., Shanghai, Lanzhou, Wuhan, Guangzhou and Yulin), we create a UFZ evaluation dataset where one large region in each city is selected and its corresponding 1-meter spatial resolution RSI is split into blocks with 256 256 pixels. A total of 4323 blocks are manually annotated by experts in remote sensing with the above 8-class land-use classification system.
